djangocms_alias/cms_toolbars.py

```diff
@@ -62,9 +61,14 @@ def add_aliases_link_to_admin_menu(self):
         if not self.request.user.has_perm('djangocms_alias.change_category'):
             return
         admin_menu = self.toolbar.get_or_create_menu(ADMIN_MENU_IDENTIFIER)
-        admin_menu.add_link_item(
-            _('Aliases'),
-            url=admin_reverse(CATEGORY_LIST_URL_NAME),
+        url = reverse(
+            "admin:{}_aliascontent_changelist".format(
+                AliasContent._meta.app_label
```

Comment:
The old url should be kept as: admin_reverse(CATEGORY_LIST_URL_NAME), this would be breaking what was designed for the urls to be configured in one place. We may be able to add additional attributes to navigate to the changelist with the category preselected, we will lose where to add that category if we remove the category constant. Also this PR has made the change to the value: https://github.com/django-cms/djangocms-alias/pull/107/files#diff-ddf7c261cd48ed07f79c9984797aaf3edc947e6fa5f976ffc78314bc5c6768f7R7